SharedCode/DI/DI.cs

```diff
@@ -27,6 +26,9 @@ public static void RegisterDIs()
 			container.RegisterType<IPokemonController, PokemonController>();
 			container.RegisterType<IPokemonDetailController, PokemonDetailController>();
 
+			// Managers
+			container.RegisterType<IDatabaseManager, DatabaseManager>();
```

Comment:
Is there a reason you are registering the database manager here and in the android/ios files?